core/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -19,7 +19,7 @@ import logger.LazyLogging
 import scala.collection.mutable
 
 private[chisel3] class Namespace(keywords: Set[String]) {
-  private val names = collection.mutable.HashMap[String, Long]()
+  private[chisel3] val names = collection.mutable.HashMap[String, Long]()
```

Comment:
Does this need to be made package private? I don't see any uses of it

Reply:
Not anymore, fixed